bin/faucet/src/main.rs

```diff
@@ -107,46 +93,36 @@ async fn run_faucet_command(cli: Cli) -> anyhow::Result<()> {
         Command::Start { config } => {
             let config: FaucetConfig =
                 load_config(config).context("failed to load configuration file")?;
-            let faucet_state = FaucetState::new(config.clone()).await?;
-
-            info!(target: COMPONENT, %config, "Initializing server");
-
-            let app = Router::new()
-                .route("/", get(get_index_html))
-                .route("/index.js", get(get_index_js))
-                .route("/index.css", get(get_index_css))
-                .route("/background.png", get(get_background))
-                .route("/favicon.ico", get(get_favicon))
-                .route("/get_metadata", get(get_metadata))
-                .route("/get_tokens", post(get_tokens))
-                .layer(
-                    ServiceBuilder::new()
-                        .layer(TraceLayer::new_for_http())
-                        .layer(SetResponseHeaderLayer::if_not_present(
-                            http::header::CACHE_CONTROL,
-                            HeaderValue::from_static("no-cache"),
-                        ))
-                        .layer(
-                            CorsLayer::new()
-                                .allow_origin(tower_http::cors::Any)
-                                .allow_methods(tower_http::cors::Any)
-                                .allow_headers([header::CONTENT_TYPE]),
-                        ),
-                )
-                .with_state(faucet_state);
-
-            let socket_addr = config
-                .endpoint
-                .socket_addrs(|| None)?
-                .into_iter()
-                .next()
-                .with_context(|| format!("no sockets available on {}", config.endpoint))?;
-            let listener =
-                TcpListener::bind(socket_addr).await.context("failed to bind TCP listener")?;
-
-            info!(target: COMPONENT, endpoint = %config.endpoint, "Server started");
 
-            axum::serve(listener, app).await.unwrap();
+            let mut rpc_client =
+                RpcClient::connect_lazy(&config.node_url).context("failed to create RPC client")?;
+            let account_file = AccountFile::read(&config.faucet_account_path)
+                .context("failed to load faucet account from file")?;
+
+            let faucet = Faucet::load(account_file, &mut rpc_client).await?;
+
+            // Maximum of 100 requests in-queue at once. Overflow is rejected for faster feedback.
+            let (tx_requests, rx_requests) = mpsc::channel(100);
```

Comment:
In future might be something we want configurable so as to update without creating a release

Reply:
Yeah I was lazy :/ I want to replace the config file with cli args, but that would be a breaking change. And then its easier to add this in.